var/spack/repos/builtin/packages/mofem-fracture-module/package.py

```diff
@@ -31,13 +31,23 @@ class MofemFractureModule(CMakePackage):
 
     maintainers = ['likask']
 
-    version('develop', branch='develop')
+    version('0.9.50', tag='v0.9.50')
+    version('0.9.49', tag='v0.9.49')
+    version('0.9.48', tag='v0.9.48')
+    version('0.9.47', tag='v0.9.47')
+    version('0.9.46', tag='v0.9.46')
+    version('0.9.45', tag='v0.9.45')
+    version('0.9.44', tag='v0.9.44')
     version('0.9.42', tag='v0.9.42')
+    version('develop', branch='develop')
```

Comment:
I prefer to put `develop` at the top since it's the newest version, but that's not a hard requirement.

Reply:
Good point, fix on the way.